src/plugins/discover/public/application/helpers/popularize_field.ts

```diff
@@ -30,7 +30,8 @@ async function popularizeField(
     return;
   }
 
-  field.count++;
+  field.count = 1 + (field.count || 0);
```

Comment:
I think the code might be a bit cleaner if we make sure this is always an integer

Reply:
Made it always to be integer.